tests/extras/datasets/spark/test_spark_dataset.py

```diff
@@ -528,7 +539,7 @@ def test_versioning_existing_dataset(
     sys.platform.startswith("win"), reason="DBFS doesn't work on Windows"
 )
 class TestSparkDataSetVersionedDBFS:
-    def test_load_latest(  # noqa: too-many-arguments
+    def test_load_latest(  # pylint: disable=too-many-arguments
```

Comment:
Why are these `pylint: disable` again? I thought with the move to `ruff` they were all changed to `noqa`?

Reply:
Good catch, some of these files I copied over from previous PR attempting Python 3.11, https://github.com/kedro-org/kedro/pull/2417. This PR was a continuation of that it seems I copied some outdated files.

Reply:
Fixed